Python_prog/addition.py

```diff
@@ -0,0 +1,5 @@
+# program to take two number as input and print sum of it
```

Comment:
Always follow good coding practices, and keep space around the operator. It improves readability of the code.